style.css

```diff
@@ -0,0 +1,131 @@
+* {
+  font-family: "Poppins", sans-serif;
+  padding: 0;
+  margin: 0;
+}
+
+.logo {
+  color: #344563;
+}
+
+body {
+  width: fit-content;
+  background-color: #e5e5e5;
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  overflow-x: hidden;
+}
+
+header {
+  background-color: white;
+  width: 100%;
+}
+
+.backgroundimg {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-image: url("/image/background.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-color: white;
+  border-radius: 0 0 0 120px;
+  height: 680px;
+  width: 100%;
+}
```

Comment:
- [ ] Please look at my screenshot below and consider removing the CSS styles that are crossed out. It will improve your current design. Plus, I added padding to it.

![image](https://user-images.githubusercontent.com/84156714/185694410-d1028de8-dbb9-4dfd-adf3-fcc96f5cb0b7.png)